tox.ini

```diff
@@ -1,12 +1,21 @@
 [tox]
-envlist = tests, flake8
+envlist = tests, marionette, flake8
 
 [testenv]
 passenv = DISPLAY PYTEST_ADDOPTS
 deps =
+    mock
+    pytest==3.0.4
+    pytest-selenium
+    pytest-xdist==1.15.0
```

Comment:
Using the environment suggestion from above, you can simply add the py27-marionette and py35-marionette dependency by adding:

```ini
deps =
    mock
    pytest==3.0.4
    pytest-selenium
    pytest-xdist==1.15.0
    py{27,35}-marionette: marionette_driver==2.1.0
```

See http://tox.readthedocs.io/en/latest/config.html#complex-factor-conditions for more information on this feature.